packages/core/test/test_bed_spec.ts

```diff
@@ -229,6 +229,64 @@ describe('TestBed', () => {
     expect(hello.nativeElement).toHaveText('Hello injected World !');
   });
 
+  it('allow to override a provider defined via ModuleWithProviders (using TestBed.overrideProvider)',
```

Comment:
Nit: I know this is consistent with the rest of the file, but it's nice when the test desc reads like a sentence (same below)

```suggestion
  it('should allow overriding a provider defined via ModuleWithProviders...', () => {
```